packages/core/assets/template/android/build.gradle

```diff
@@ -29,38 +24,10 @@ buildscript {
     }
     dependencies {
         classpath("org.jetbrains.kotlin:kotlin-gradle-plugin:<%- android.gradle?.projectGradle?.kotlinVersion || "1.6.0" %>")
-        classpath("com.android.tools.build:gradle:7.2.1")
+        classpath("com.android.tools.build:gradle:7.3.1")
         classpath("com.facebook.react:react-native-gradle-plugin")
-        classpath("de.undercouch:gradle-download-task:5.0.1")
         <% android.gradle?.projectGradle?.dependencies?.forEach(function(it) { %>
         <%- it %>
         <% }) %>
-        // NOTE: Do not place your application dependencies here; they belong
-        // in the individual module build.gradle files
-    }
-}
-
-allprojects {
```

Comment:
shouldn't remove the object completely - just it's contents + we need ejs content